src/components/ForecastDetails.js

```diff
@@ -1,53 +0,0 @@
-import React from "react";
-import PropTypes from "prop-types";
-import "../styles/ForecastDetails.css";
-function ForecastDetails({ forecast }) {
-  const formattedDate = new Date(forecast.date).toDateString();
```

Comment:
I would destructure each of these on one line like this:
`const { date, description, wind, temperature } = forecast`
The code here looks a little messy because of all these variables